apps/server/src/setup.ts

```diff
@@ -71,14 +72,15 @@ export const currentDirectory = dirname(__dirname);
 const testDbStartDirectory = isTest ? '../' : getAppDataPath();
 export const externalsStartDirectory = isProduction ? getAppDataPath() : join(currentDirectory, 'external');
 
-const lastLoadedProjectConfigPath = join(getAppDataPath(), 'config.json');
+export const lastLoadedProjectConfigPath = join(getAppDataPath(), 'config.json');
 
 let lastLoadedProject;
 
 try {
   lastLoadedProject = JSON.parse(fs.readFileSync(lastLoadedProjectConfigPath, 'utf8')).lastLoadedProject;
 } catch {
   if (!isTest) {
+    ensureDirectory(getAppDataPath());
```

Comment:
good catch, there starts to be a lot here, do you think we should have a setup function that makes sure we have everything we need?

Reply:
yeah imo that would be a good idea

Reply:
I will make a note, maybe this is something you could help with later 😅